apps/team-awesome-web/jest.config.ts

```diff
@@ -1,11 +1,12 @@
 /* eslint-disable */
 export default {
-  displayName: 'team-awesome-web',
-  preset: '../../jest.preset.js',
+  displayName: "team-awesome-web",
+  preset: "../../jest.preset.js",
   transform: {
-    '^(?!.*\\.(js|jsx|ts|tsx|css|json)$)': '@nrwl/react/plugins/jest',
-    '^.+\\.[tj]sx?$': ['babel-jest', { presets: ['@nrwl/next/babel'] }],
+    "^(?!.*\\.(js|jsx|ts|tsx|css|json)$)": "@nrwl/react/plugins/jest",
+    "^.+\\.[tj]sx?$": ["babel-jest", { presets: ["@nrwl/next/babel"] }],
   },
-  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx'],
-  coverageDirectory: '../../coverage/apps/team-awesome-web',
+  moduleFileExtensions: ["ts", "tsx", "js", "jsx"],
+  coverageDirectory: "../../coverage/apps/team-awesome-web",
```

Comment:
Wat is dit?

Reply:
Linting, only change was the replacement of quotation marks.